packages/react/README.md

```diff
@@ -120,7 +125,8 @@ function App() {
 }
 
 function Page() {
-  const newMessage = useBooleanFlagValue('new-message', false);
+  // Use the "query-style" flag evaluation hook.
+  const { value: newMessage } = useFlag('new-message', true);
```

Comment:
This README promotes the new hook first, which I think is good, LMK what you think.

Reply:
Question: Should we add why you might want to use this hook over others? Does this new hook render the others as deprecated or to-be deprecated?

Reply:
> Question: Should we add why you might want to use this hook over others? Does this new hook render the others as deprecated or to-be deprecated?

The reason is just for the react-style return object that enabled destructuring-assignment of a bunch of useful props, and the fact that it's totally generic. It's really just react-flavored sugar.

Reply:
That makes sense, I just worry about devs not knowing if this hook is recommended over all of the others, when to use this hook, or if the other hooks are going away.

Reply:
I've tried to outline this in the README, bit I could certain add more inline doc that makes it clear both APIs are fine to use. How does that sound?

Reply:
Done, see: https://github.com/open-feature/js-sdk/pull/897/commits/6938f9bdb51e5023503d2fbe4d9fa9fc89d8fd17